notebooks/tutorial.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Process calcium imaging data with DataJoint Elements\n",
+    "\n",
+    "This notebook will walk through processing two-photon calcium imaging data collected\n",
+    "from ScanImage and processed with Suite2p.\n",
+    "\n",
+    "The DataJoint Python API and Element Calcium Imaging offer a lot of features to support collaboration, automation, reproducibility, and visualization.\n",
+    "\n",
+    "For more information on these topics, please visit our documentation: \n",
+    " \n",
+    "- [DataJoint Core](https://datajoint.com/docs/core/): General principles\n",
+    "\n",
+    "- DataJoint [Python](https://datajoint.com/docs/core/datajoint-python/) and\n",
+    "  [MATLAB](https://datajoint.com/docs/core/datajoint-matlab/) APIs: in-depth reviews of\n",
+    "  specifics\n",
+    "\n",
+    "- [DataJoint Element Calcium Imaging](https://datajoint.com/docs/elements/element-calcium-imaging/):\n",
+    "  A modular pipeline for calcium imaging analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datajoint as dj"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dj.conn()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define schema prefixes and root data directory\n",
+    "\n",
+    "+ If we set prefix to `neuro_`, every schema created with the current workflow will start with `neuro_`, e.g. `neuro_lab`, `neuro_scan`, `neuro_imaging`, etc.\n",
+    "\n",
+    "+ The example dataset is attached to this GitHub Codespace environment, so the root directory is `/workspaces/workflow-calcium-imaging/example_data`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dj.config[\"custom\"] = {\"database.prefix\": \"neuro_\",\n",
+    "                       \"imaging_root_data_dir\": \"/workspaces/workflow-calcium-imaging/example_data\"}"
```

Comment:
```suggestion
    "dj.config[\"custom\"] = {\"database.prefix\": \"neuro_\",\n",
    "                       \"imaging_root_data_dir\": \"../example_data\"}"
```

Reply:
Discussed option to change to environment.  Will update in subsequent pull request.